src/python/fuzzing/strategy.py

```diff
@@ -38,6 +38,8 @@
 FORK_STRATEGY = Strategy(name='fork', probability=0.50, manually_enable=False)
 MUTATOR_PLUGIN_STRATEGY = Strategy(
     name='mutator_plugin', probability=0.50, manually_enable=True)
+RADAMSA_MUTATOR_PLUGIN_STRATEGY = Strategy(
+    name='radamsa_mutator_plugin', probability=1.0, manually_enable=False)
```

Comment:
1.0 means 100%, you probably meant `0.01` for starters

Reply:
It was 1.0 while testing. I agree that 0.01 is where we want to start it.